app/models/services/pubmed/http.rb

```diff
@@ -1,33 +1,39 @@
 class Pubmed
   class Http
-    def self.default_parameters
+    def self.eutils_params
       {
         db: 'pubmed',
         retmode: 'xml',
         retmax: 20
       }
     end
 
-    def self.esearch options = {}
-      uri = generate_uri("/entrez/eutils/esearch.fcgi", default_parameters.merge(options))
-
-      Net::HTTP.get_response(uri)
+    def self.id_params
+      {
+        tool: 'projectcredo',
+        email: 'accounts@projectcredo.com'
+      }
     end
```

Comment:
Note that these identifiers are things that the various Pubmed services expect to have for their own instrumentation purposes.